frontend/src/hex/HexView.svelte

```diff
@@ -265,123 +273,140 @@
     return info;
   }
 
-  let dataSearchResults = {};
-
-  // React to local data searches
-  $: {
-    const localDataSearchResults = dataSearchResults;
-
-    if (
-      localDataSearchResults.matches?.length > 0 &&
-      (localDataSearchResults.index || localDataSearchResults.index === 0)
-    ) {
-      dataLenPromise.then((dataLength) => {
-        $scrollY.top =
-          localDataSearchResults.matches[localDataSearchResults.index][0] /
-          dataLength;
-      });
-    }
+  function resetResource() {
+    dataSearchResults.matches = undefined;
+    currentPosition = 0;
   }
 
-  async function searchHex(query, options) {
-    return await $selectedResource.search_data(query, options);
+  function refreshHeight() {
+    hexDisplay = document.getElementById("scrollable");
```

Comment:
I think we talked about having this be set by `bind:this`?

Reply:
Woops, i added this, but missed this re-assignment.